src/LanguageServer/Test/GoToDefinitionTests.cs

```diff
@@ -235,6 +235,35 @@ class A(object):
             reference.uri.Should().Be(modPath);
         }
 
+        [TestMethod, Priority(0)]
+        public async Task GotoModuleSourceFromRelativeImport() {
+            var modAPath = TestData.GetTestSpecificUri("a.py");
+            var modBPath = TestData.GetTestSpecificUri("b.py");
+
+            var root = TestData.GetTestSpecificRootUri().AbsolutePath;
+            await CreateServicesAsync(root, PythonVersions.LatestAvailable3X);
```

Comment:
We should probably add overload to the `CreateServicesAsync` - `TestData.GetTestSpecificRootUri().AbsolutePath` is used in majority of tests.

Reply:
Sure